src/docfx/Errors.cs

```diff
@@ -129,6 +129,9 @@ public static Error NullArrayValue(SourceInfo? source, string name)
             /// Behavior: ❌ Message: ❌
             public static Error ViolateSchema(SourceInfo? source, string message)
                 => new Error(ErrorLevel.Error, "violate-schema", $"{message}", source);
+
+            public static Error UnexpectedType(FormattableString message)
```

Comment:
Reuse the existing `Errors.JsonSchema.UnexpectedType` error

Reply:
OK.